README.md

```diff
@@ -1,22 +1,19 @@
 <div align="center">
-	<h1>The Nexus Mods app</h1>
+	<h1>The Nexus Mods App</h1>
```

Comment:
The lower case `a` in `app` is a deliberate choice.
It used to be `Nexus Mods App`, but we've moved to lowercase at some point last year.